sgp30/sgp30_example_usage.c

```diff
@@ -61,10 +61,17 @@ int main(void) {
 
     /* Busy loop for initialization. The main loop does not work without
      * a sensor. */
-    while (sgp30_probe() != STATUS_OK) {
+    int16_t probe;
+    while ((probe = sgp30_probe()) != STATUS_OK && probe != STATUS_UNKNOWN_FEATURE_SET) {
```

Comment:
it's tempting, but don't use assign and test - go with `while(1) {}` or `do {} while();`